src/index.html

```diff
@@ -7,8 +7,28 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="site-header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="logo" >
+      </a>
+      
+      <nav class="site-nav">
+        <ul>
+          <li class="is-active"><a href="#0">apple</a></li>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list>
          <li class="nav__item is-active">
            <a class="nav__link" href="#0">apple</a></li>
```
Створи для всіх елементів класи і не використовуй селектори по тегам

Reply:
Зробила виправлення по вашим коментарям і все стало ще гірше